src/Utils.js

```diff
@@ -187,6 +200,9 @@ export const SELECTOR = {
 };
 
 export function selectorType(selector) {
+  if (isPseudoClassSelector(selector)) {
+    throw selectorError(selector, 'pseudo-class');
```

Comment:
It's hyphenated in [the spec](https://www.w3.org/TR/CSS21/selector.html#pseudo-class-selectors).